ci-operator/config/Azure/ARO-HCP/Azure-ARO-HCP-main__periodic.yaml

```diff
@@ -46,19 +51,16 @@ tests:
 - as: delete-expired-integration-resource-groups
   cron: 7,37 * * * *
   steps:
-    cluster_profile: aro-hcp-int
```

Comment:
@stevekuznetsov was there a particular reason to make the deletion job consume from the quota?